app/client/src/components/shared/WaterbodyInfo.js

```diff
@@ -83,13 +83,15 @@ const modifiedTableStyles = css`
   td {
     overflow-wrap: anywhere;
     hyphens: auto;
+    white-space: pre-wrap;
 
     :first-of-type {
       padding-left: 0;
     }
 
     :last-of-type {
       padding-right: 0;
+      text-align: center;
```

Comment:
```suggestion
```

Similar to above.

Reply:
Sounds good, it should be back to where it was now.